tests/www/views/test_views_tasks.py

```diff
@@ -66,7 +66,7 @@ def reset_dagruns():
 
 
 @pytest.fixture(autouse=True)
-def init_dagruns(app, reset_dagruns):
```

Comment:
What was this used for?

Reply:
This basically initializes the DAG runs with pre-made execution dates and specific states. Basically for every test, it runs for every test in the module.